web/src/data/songs_of_the_week.js

```diff
@@ -14,9 +14,12 @@ class SongOfTheWeek {
 
 const attributors = {
   riley: {
-    name: "Riley",
+    name: "Riley C",
     picture: RileyHeadshot,
   },
+  makenzie: {
+    name: "Makenzie S",
+  },
 };
```

Comment:
can I be a song recommender?

Reply:
Yes. Fill out https://docs.google.com/forms/d/e/1FAIpQLSeR3MAXz_-eDNrGjF6SxajrbrO2oxQhksPFDSoysj2iiD_oVA/viewform